lib/usd/ui/layerEditor/stringResources.h

```diff
@@ -47,61 +47,61 @@ MStringResourceId create(const char* key, const char* value);
+const auto kSaveLayerWarnMsg             { create("kSaveLayerWarnMsg", "Saving edits to ^1s will overwrite your file.") };
+const auto kSaveStage                    { create("kSaveStage", "Save Stage") };
+const auto kSaveStages                   { create("kSaveStages", "Save Stage(s)") };
+const auto kSaveXStages                  { create("kSaveXStages", "Save ^1s Stage(s)") };
+const auto kToSaveTheStageSaveAnonym     { create("kToSaveTheStageSaveAnonym", "To save the ^1s stage(s), save the following ^2s anonymous layer(s).") };
+const auto kToSaveTheStageSaveFiles      { create("kToSaveTheStageSaveFiles", "To save the ^1s stage(s), the following existing file(s) will be overwritten.") };
+const auto kUsedInStagesTooltip          { create("kUsedInStagesTooltip", "<b>Used in Stages</b>: ") };
+
+const auto kSetLayerAsTargetLayerTooltip { create("kSetLayerAsTargetLayerTooltip", "Set layer as target layer. Edits are added to the target layer.") };
+const auto kUsdSaveFileFormat            { create("kUsdSaveFileFormat", "Save .usd File Format") };
+const auto kUsdLayerIdentifier           { create("kUsdLayerIdentifier", "USD Layer identifier: ^1s") };
+const auto kUsdStage                     { create("kUsdStage", "USD Stage:") };
 
 const auto kSaveAnonymousLayersErrorsTitle { create("kSaveAnonymousLayersErrorsTitle", "Save All Layers Error")};
 const auto kSaveAnonymousLayersErrorsMsg { create("kSaveAnonymousLayersErrorsMsg", "Errors were encountered while saving layers.  Check Script Editor for details.")};
-const auto kSaveAnonymousLayersErrors { create("kSaveAnonymousLayersErrors", "Layer ^1s could not be saved to: ^2s")};
+const auto kSaveAnonymousLayersErrors    { create("kSaveAnonymousLayersErrors", "Layer ^1s could not be saved to: ^2s")};
 const auto kSaveAnonymousConfirmOverwriteTitle { create("kSaveAnonymousConfirmOverwriteTitle", "Confirm Overwrite")};
 const auto kSaveAnonymousConfirmOverwrite { create("kSaveAnonymousConfirmOverwrite", "^1s file(s) already exist and will be overwritten.  Do you want to continue?")};
 
-const auto kSaveLayerUsdFileFormatAnn{ create("kSaveLayerUsdFileFormatAnn", "Select whether the .usd file is written out in binary or ASCII. You can save a file in .usdc (binary) or .usda (ASCII) format. Manually entering a file name with an extension overrides the selection in this drop-down menu.") };
-const auto kSaveLayerUsdFileFormatSbm{ create("kSaveLayerUsdFileFormatSbm", "Select whether the .usd file is written out in binary or ASCII") };
+const auto kSaveLayerUsdFileFormatAnn    { create("kSaveLayerUsdFileFormatAnn", "Select whether the .usd file is written out in binary or ASCII. You can save a file in .usdc (binary) or .usda (ASCII) format. Manually entering a file name with an extension overrides the selection in this drop-down menu.") };
+const auto kSaveLayerUsdFileFormatSbm    { create("kSaveLayerUsdFileFormatSbm", "Select whether the .usd file is written out in binary or ASCII") };
 
```

Comment:
Did you actually change anything in this file? It looks like all whitespace changes to me. If so was it you or because of clang-format?

Reply:
Oh, I had some changes but they were removed, its only clang format I guess